lib/bolt/cli.rb

```diff
@@ -575,10 +575,15 @@ def show_task(task_name)
       outputter.print_task_info(pal.get_task(task_name))
     end
 
+    # Filters a list of content by matching substring.
+    #
+    private def filter_content(content, filter)
```

Comment:
Why are we filtering content twice? It seems like just filtering it in PAL should work.

Reply:
It will get filtered twice regardless, though I don't really care _where_ that happens. Leaving the `--filter`... filtering in the CLI made sense per Nick's comment above about it being a separate concept entirely. 🤷

Reply:
Restricting the content to only what belongs to the project is an API level operation. Picking a subset of that to view is a UI operation. So I think it's helpful to keep them separate.

Reply:
Yeah sorry I didn't realize we had a CLI filter option.